packages/app/src/reducers/config.js

```diff
@@ -0,0 +1,15 @@
+export const SET_CONFIG = 'SET_CONFIG'
+
+export const DEFAULT_CONFIG = null
+
+export default (state = DEFAULT_CONFIG, action) => {
+    switch (action.type) {
+        case SET_CONFIG: {
+            return action.value
+        }
+        default:
+            return state
+    }
+}
+
+export const sGetVisConfig = state => state.visConfig
```

Comment:
This selector doesn't seem to be used anywhere.

Reply:
True, since cypress reads the store through `getReduxState` rather than selectors

Reply:
Removed